### ISSUE TITLE
Indeterminate Progress Bar

### ISSUE THREAD

**Issue by eygraber** [VictorBjelkholm/ngProgress#61]
I don't know if you take feature requests here, but I think an indeterminate progress bar would be great. Something along the lines of what Android does (http://i.stack.imgur.com/m1aGNm.png - the blue lines at the top are animated to move left to right in an infinite loop).

**Comment by VictorBjelkholm** [comment]
Yes, this is a interesting feature that I will be looking into. If anyone have any ideas or want to try and implement it, let me know.
